test/qa/lib/settings.py

```diff
@@ -1,86 +1,104 @@
 import sh
 
 
+class Settings:
+    def __init__(self):
+        output = sh.nordvpn("settings").strip(" \r-\n")
+
+        self.settings={}
+        previous_key=""
+        for line in output.split("\n"):
+            values = line.split(":")
+            if len(values) == 2:
+                previous_key = values[0].lower().strip()
+                self.settings[previous_key] = values[1].strip()
+            elif len(previous_key) > 0:
+                # for allow list the values are on a different line
+                self.settings[previous_key] += line.strip() + " "
+
+    def get(self, key: str) -> str:
+        key = key.lower()
+        return self.settings.get(key, "")
+
+    # convenient to get a value from settings only once
+    @staticmethod
+    def get_value(key: str) -> str:
+        return Settings().get(key)
+
+
 def get_server_ip() -> str:
     """Returns str with IP Address of the server from `nordvpn status`, that NordVPN client is currently connected to."""
     return sh.nordvpn.status().split('\n')[3].replace('IP: ', '')
 
 
 def get_current_connection_protocol():
```

Comment:
Is there a good reason for such a static method? It only allows to call `Settings.get_value("xxx")` instead of `Settings().get("xxx")`, which actually requires more characters to write.